### PR TITLE
Adding language about preprints

### PR DESCRIPTION
**Pull request by arfon**
Clarifying our position on preprints.

### CODE REVIEW

**Comment by arfon** [comment]
/ cc @labarba @danielskatz @kyleniemeyer

File: docs/submitting.md [hunk 1]
```diff
@@ -220,6 +220,12 @@ Submission is as simple as:
 
 There are no fees for submitting or publishing in JOSS. You can read more about our [cost and sustainability model](http://joss.theoj.org/about#costs).
 
+## Preprint Policy
+
+Authors are welcome to submit their papers to a preprint server such as [arXiv](https://arxiv.org/), [bioRxiv](https://www.biorxiv.org/), [SocArXiv](https://socopen.org/), [PsyArXiv](https://psyarxiv.com/) at any point during the submission and review process.
```
Comment (danielskatz) [inline docs/submitting.md]: Authors are welcome to submit their papers to a preprint server (e.g., [arXiv](https://arxiv.org/), [bioRxiv](https://www.biorxiv.org/), [SocArXiv](https://socopen.org/), [PsyArXiv](https://psyarxiv.com/)) at any point during the submission and review process.

File: docs/submitting.md [hunk 2]
```diff
@@ -222,7 +222,7 @@ There are no fees for submitting or publishing in JOSS. You can read more about
 
 ## Preprint Policy
 
-Authors are welcome to submit their papers to a preprint server such as [arXiv](https://arxiv.org/), [bioRxiv](https://www.biorxiv.org/), [SocArXiv](https://socopen.org/), [PsyArXiv](https://psyarxiv.com/) at any point during the submission and review process.
+Authors are welcome to submit their papers to a preprint server (e.g., [arXiv](https://arxiv.org/), [bioRxiv](https://www.biorxiv.org/), [SocArXiv](https://socopen.org/), [PsyArXiv](https://psyarxiv.com/)) at any point during the submission and review process.
```
Comment (danielskatz) [inline docs/submitting.md]: I would like to somehow indicate that these are just some options, which is what I thought the eg and parens would do, but I'm not sure that succeeded.  I know it's not correct grammatically, but perhaps add ", etc." at the end of the list?

**Comment by kyleniemeyer** [comment]
If you have "e.g.," then "etc." is redundant—only need one or the other.

**Comment by danielskatz** [comment]
Yes, I know, but that doesn’t quite send the right message, even though the grammar is correct. Maybe drop the eg?

**Pull request merged**